assets/src/onboarding-wizard/index.js

```diff
@@ -90,13 +95,24 @@ Providers.propTypes = {
 domReady( () => {
 	const root = document.getElementById( APP_ROOT_ID );
 
-	if ( root ) {
-		render(
-
-			<Providers>
-				<SetupWizard closeLink={ CLOSE_LINK } finishLink={ FINISH_LINK } appRoot={ root } />
-			</Providers>,
-			root,
-		);
+	if ( ! root ) {
+		return;
 	}
+
+	const errorHandler = ( error ) => {
+		// Handle only own errors.
+		if ( error?.filename?.match( /amp-onboarding-wizard(\.min)?\.js/ ) ) {
```

Comment:
This would be more appropriate since we're not making use of the matches:

```suggestion
		if ( /amp-onboarding-wizard(\.min)?\.js/.test( error?.filename ) ) {
```